explainer.md

```diff
@@ -16,6 +16,7 @@ The Payment Request API can be used, with [a minor modification](https://github.
 *   Querying the details (e.g., name, description, regional price) of digital items from the store backend.
     *   Note: Even though the web app developer is ultimately responsible for configuring these items on the server, and could therefore keep track of these without an API, it is important to have a single source of truth, to ensure that the price of items displayed in the app exactly matches the prices that the user will eventually be charged, especially as prices can differ by region, or change at planned times (such as when sale events begin or end).
 *   Consuming or acknowledging purchases. Digital stores typically do not consider a purchase finalized until the client acknowledges the purchase through a separate API call. This acknowledgment is supposed to be performed once the client “activates” the purchase inside the app.
+*   Checking the digital items currently owned by the user (e.g., active subscriptions and non-consumed one-time purchases).
```

Comment:
This is the first mention in the document of the terms "subscriptions, "non-consumed", and "one-time". Perhaps the text in parentheses here could be removed?

Reply:
"Consuming" is mentioned in the bullet above. I'll remove "one-time".
We don't mention subscriptions anywhere in DigitalGoods but it's sounding like they may be an important thing to call out because they have this new property of remaining "owned" for a certain duration after being purchased/consumed/acknowledged (only noticeable through this new API method).

Reply:
Hmm, I still think the parenthesised text isn't really necessary to validate the necessity of this method (querying purchasable things and querying what has been purchased go naturally together). I think it's also preferable to no need to mention "subscriptions" in the API since that's really an implementation detail that the API enables, not a first class concept in the API.

Reply:
Ok, removed the parenthesised bit.